.travis.yml

```diff
@@ -1,5 +1,23 @@
-script:
-- docker build --target test --tag my-test-image .
-- docker run my-test-image ./todo_app/tests/todo_view_model_test.py
-- docker run my-test-image ./todo_app/tests/test_client.py
-- docker run -e API_KEY -e API_TOKEN -e BOARD_NAME -e MEMBERS_NAME -e TO_DO_LIST_ID -e DOING_LIST_ID -e DONE_LIST_ID  -e FLASK_APP -e FLASK_ENV -e SECRET_KEY my-test-image  todo_app/tests_e2e
+language: python
+
+services: docker
+
+jobs: 
+  include:
+    - stage: test
+      script:
+      - docker build --target test --tag my-test-image .
+      - docker run my-test-image ./todo_app/tests/todo_view_model_test.py
+      - docker run my-test-image ./todo_app/tests/test_client.py
+      - docker run -e API_KEY -e API_TOKEN -e BOARD_NAME -e MEMBERS_NAME -e TO_DO_LIST_ID -e DOING_LIST_ID -e DONE_LIST_ID  -e FLASK_APP -e FLASK_ENV -e SECRET_KEY my-test-image  todo_app/tests_e2e
```

Comment:
MEMBERS_NAME is not used by your app at all. 

And BOARD_NAME, TODO_LIST_ID, DOING_LIST_ID, DONE_LIST_ID are not used by the e2e test - it creates its own temporary board.

Reply:
Removed unneccesary env variables